opendebates/settings.py

```diff
@@ -8,6 +8,39 @@
 BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
 FIXTURE_DIRS = [os.path.join(BASE_DIR, 'fixtures')]
 
+SITE_ID = 1
+SITE_DOMAIN = os.environ.get("SITE_DOMAIN", "127.0.0.1:8000")
+SITE_DOMAIN_WITH_PROTOCOL = os.environ.get("SITE_PROTOCOL", "http://") + SITE_DOMAIN
+
+SITE_THEMES = {
+    'florida': {
+        "HASHTAG": "FLOpenDebate",
+        "HEADER_TITLE": _("WELCOME TO THE\nFLORIDA OPEN DEBATE"),
```

Comment:
(I think we'd need to remove the `django.middleware.locale.LocaleMiddleware` from MIDDLEWARE_CLASSES in order to use the approach I'm proposing -- so everyone just gets the same "translation", per `settings.LANGUAGE_CODE`, without any reference to the accept-language being sent by the browser.)